src/components/Share/Share.css

```diff
@@ -0,0 +1,60 @@
+.Share {
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: 0;
+  margin: 0.5rem;
+  line-height: 1.5rem;
+
+  h1 {
+    display: inline-block;
+    font-size: 1.5rem;
+    font-weight: 100;
+
+    strong {
+      font-weight: 600;
+    }
+  }
+
+  input {
+    position: absolute;
+    left: -99999px;
```

Comment:
`<input>`  needs to be _visible_ for this trick to work, and by _visible_ it means it cannot be `display: none` or `visibility: hidden`. `opacity: 0` and/or `width: 0; height: 0` works, but does not look any less hacky. Actually, kicking elements off screen like this is a common trick used in front end development for years.